app.js

```diff
@@ -0,0 +1,23 @@
+const express = require('express');
+const {sendTopics} = require('./controllers/news.controllers')
+
+const app = express()
+
+app.get('/api/topics',sendTopics)
+
+
+
+app.use((err, req, res, next)=>{
+if(err.code ==='22P02'|| err.code === '42703') {
+    res.status(400).send({msg:'Bad Request'})
+}
```

Comment:
I totally understand the temptation to put these codes in, but you don't need them for this task. I would try to stick to just adding code that you need for the task you are on in the future 👍